demo/README.md

```diff
@@ -0,0 +1,34 @@
+# Demos for Membrane Video Compositor Plugin
+
+<!-- [![Hex.pm](https://img.shields.io/hexpm/v/membrane_video_compositor_plugin.svg)](https://hex.pm/packages/membrane_video_compositor_plugin)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_video_compositor_plugin) -->
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/membraneframework-labs/membrane_video_compositor_plugin/tree/master.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/membraneframework-labs/membrane_video_compositor_plugin/tree/master)
+
+Set of demos for Membrane video compositor plugin. 
+
+
+## Usage
+
+Run demos with the command line, for example:
+
+`IMPL=ffmpeg SINK=file  mix run lib/h264_pipeline_script.exs`
+
+### Possible options:
+- `IMPL=` - implementation of video compositor 
+  - `ffmpeg`
+  - `nx`
+- `SINK=` - result destination
+  - `file` - create an output file with the corresponding video format
+  - `play` - play result in real-time, using SDL video player
```

Comment:
`System` module allows you to use argv easily, but I leave it up to you

Reply:
I leave it as it is, right now